### ISSUE TITLE
TypeError: Can't instantiate abstract class Primary with abstract methods update_menus

### ISSUE THREAD

**Issue by g4570n** [xgi/castero#58]
I just updated to castero 0.6.0 and I have this error when I want to start it:

```
Traceback (most recent call last):
                                    File "/home/g4570n/bin/castero", line 7, in <module>
            from castero.__main__ import main
                                               File "/home/g4570n/.local/lib/python3.5/site-packages/castero/__main__.py", line 113, in <module>
                                                                    main()
                                                                            File "/home/g4570n/.local/lib/python3.5/site-packages/castero/__main__.py", line 86, in main
            display = Display(stdscr, database)
                                                 File "/home/g4570n/.local/lib/python3.5/site-packages/castero/display.py", line 99, in __init__
                                                                    self._load_perspectives()
               File "/home/g4570n/.local/lib/python3.5/site-packages/castero/display.py", line 163, in _load_perspectives
                                             inst = p_cls(self)
                                                               TypeError: Can't instantiate abstract class Primary with abstract methods update_menus
```

**Comment by xgi** [comment]
I think I figured out why this may have happened. In an earlier version of castero, I had some class files named `primary.py`, `simple.py`, etc. With 0.6.0, I renamed these to `primaryperspective.py` and `simpleperspective.py`. When you updated castero, the old files may not have been removed. The client would then try to load the old class which is no longer compatible with the new version.

To confirm this hypothesis, you could check whether `~/.local/lib/python3.5/site-packages/castero/perspectives` contains `primary.py` (I believe it will, even though it no longer exists with 0.6.0).

If this is the case, the best fix is probably just to uninstall and reinstall (`pip uninstall castero`). You won't lose any added feeds or config settings. Please let me know if this works. Thanks!

**Comment by g4570n** [comment]
I uninstalled `castero` as you told me (`pip uninstall castero`), but in the path: `~/.local/lib/python3.5/site-packages/castero/perspectives`, these files remained: `primary.py `and `queuelisting.py`, then I reinstalled `castero` (`pip install castero`), but when executing it I still had the same problem.

The solution for me was to remove manually: `primary.py` and `queuelisting.py`. Now it works! :)
